doc/Navigation/HowTo-DisplayItem.md

```diff
@@ -6,138 +6,165 @@ This topic walks through how to use Navigation to display the details of an item
 
-public record AdvancedWidget(string Name, double Weight) : Widget(Name, Weight) { }
+- Change the `Navigation.Request` property value to `""`. Navigation will use the type of the selected item to determine what route to use. 
 
+- Add two additional records, `BasicWidget` and `AdvancedWidget`, that derive from `Widget`. 
 
-public Widget[] Widgets { get; } = new Widget[]
-{
-    new BasicWidget("NormalSpinner", 5.0),
-    new AdvancedWidget("HeavySpinner",50.0)
-};
-```
-
-- Copy the second page's XAML and code behind to a new third page with a different file name
-- Copy the second view model to a newly-created, third view model
-- Change constructor of both the second and third view models to accept widgets of different types
-
-- Change `ViewMap` and `RouteMap` to include the third view model
-
-```csharp
-private static void RegisterRoutes(IViewRegistry views, IRouteRegistry routes)
-{
-    views.Register(
-        new ViewMap<ShellControl, ShellViewModel>(),
-        new ViewMap<MainPage, MainViewModel>(),
-        new ViewMap<SecondPage, SecondViewModel>(Data: new DataMap<BasicWidget>()),
-        new ViewMap<ThirdPage, ThirdViewModel>(Data: new DataMap<AdvancedWidget>())
-        );
-
-    routes
-        .Register(
-            new RouteMap("", View: views.FindByViewModel<ShellViewModel>(),
+    ```csharp
+    public record Widget(string Name, double Weight) { }
+    
+    public record BasicWidget(string Name, double Weight) : Widget(Name, Weight) { }
+    
+    public record AdvancedWidget(string Name, double Weight) : Widget(Name, Weight) { }
+    ```
+- Change the `Widgets` property in `MainViewModel` to include an array of different types of widgets.
+
+    ```csharp
+    public Widget[] Widgets { get; } = new Widget[]
+    {
+        new BasicWidget("NormalSpinner", 5.0),
+        new AdvancedWidget("HeavySpinner",50.0)
+    };
+    ```
+
+- Clone the `SecondPage.xaml` and `SecondPage.xaml.cs` files, and rename the files to `ThirdPage.xaml` and `ThirdPage.xaml.cs` respectively. Make sure you also change the class name in both files from `SecondPage` to `ThirdPage`
+- Clone `SecondViewModel.cs` and rename to `ThirdViewModel.cs`. Also rename the class from `SecondViewModel` to `ThirdViewModel`
+- Change constructor of both the `SecondViewModel` and `ThirdViewModel` to accept widgets of different types
+    ```csharp
+    public class SecondViewModel
+    {
+        public string Name { get; }
+        
+        public SecondViewModel(BasicWidget widget)
+    	{
+            Name = widget.Name; 
+    	}
+    }
+    public class ThirdViewModel
+    {
+        public string Name { get; }
+        
+        public ThirdViewModel(AdvancedWidget widget)
+    	{
+            Name = widget.Name; 
+    	}
+    }
+    ```
+
+- Change `ViewMap` and `RouteMap` to include the `ThirdPage` and `ThirdViewModel`. Also update DataMap for `SecondViewModel` to be `BasicWidget` instead of `Widget`
```

Comment:
```suggestion
- Change `ViewMap` and `RouteMap` to include `ThirdPage` and `ThirdViewModel`. Also update DataMap for `SecondViewModel` to be `BasicWidget` instead of `Widget`
```